src/tracing.rs

```diff
@@ -97,14 +94,10 @@ impl FieldFormatter for FlattenFmt {
         event.record(event_record.deref_mut());
 
         if let Some(span) = current_span {
-            let extensions = span.extensions();
-            if let Some(record) = extensions.get::<fluent::Map>() {
-                event_record.update(record);
-            }
-            while let Some(span) = span.parent() {
+            for span in span.scope() {
```

Comment:
Does it traverse current and all parents?

Reply:
I don't really get why it behaves differently from `scope` but since `tracing` is poorely documented I need to check their code